dataGenerate.js

```diff
@@ -0,0 +1,38 @@
+import short from "short-uuid";
+import fs from "fs";
+
+class book {
+  constructor() {
+    this.genres = [
+      "Mystery",
+      "Science Fiction",
+      "Fantasy",
+      "Romance",
+      "Thriller",
+      "Historical Fiction",
+      "Horror",
+      "Adventure",
+      "Non-fiction",
+      "Biography",
+      "Self-help",
+    ];
+  }
+
+  generateBookData() {
+    const book = {
+      bookId: short.generate(),
+      genre: this.genres[Math.floor(Math.random() * 100) % this.genres.length],
+      price: Math.floor(Math.random() * 10000),
+    };
+    return book;
```

Comment:
these lines can be written as - 
```
return {
  bookId: short.generate(),
  genre: this.genres[Math.floor(Math.random() * 100) % this.genres.length],
  price: Math.floor(Math.random() * 10000),
}
```
No need to define an extra variable i.e. **book** if it is not used further.

Reply:
done